wolfcrypt/benchmark/benchmark.c

```diff
@@ -2842,6 +2843,10 @@ int benchmark_init(void)
         return EXIT_FAILURE;
     }
 
+#ifdef HAVE_WC_INTROSPECTION
+    printf("Math: %s\n", wc_GetMathInfo());
```

Comment:
Do we have something that will be more embedded-friendly than `printf` ?

Reply:
`printf` is fine for embedded targets.  it even works (more or less) in newlib-nano.  troubles arise with `fprintf` though, so your point is well taken.